.ladle/adapters/PlainComponentAdapter.tsx

```diff
@@ -777,4 +778,6 @@ export const PlainComponentAdapter: ComponentsContextType = {
       </div>
     )
   },
+  // eslint-disable-next-line jsx-a11y/anchor-has-content
+  Link: (props: LinkProps) => <a {...props} />,
```

Comment:
This warning is just because it isn't detecting the content provided by the spread. also it's ladle specific